% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/AsyncError.R
\docType{data}
\name{AsyncError}
\alias{AsyncError}
\alias{AsyncTaskError}
\title{Exception class for async-related errors}
\format{An object of class \code{Class} (inherits from \code{Object}) of length 1.}
\usage{
AsyncError(...)
}
\arguments{
\item{\ldots}{Passed to \code{\link[R.oo]{Exception}}}
}
\description{
Exception class for async-related errors
}
\keyword{internal}

